src/pages/blog/articles/_all-articles.js

```diff
@@ -17,9 +17,9 @@ const AllArticles = ({ article_data }) => {
                 <StyledImg src={RightArrow} height="16" width="16" />
                 <Text>All articles</Text>
             </Flex>
-            <FeaturedArticle article_data={article_data} />
+            <FirstArticle item={article_data[0]} />
             <VideoGrid m="40px 0">
-                {article_data.map((item) => {
+                {article_data.slice(1).map((item) => {
```

Comment:
Do we need to sort the articles by published date ?

Reply:
Yep that's right, we do.

Reply:
done in the next PR